Modules/Loadable/Segmentations/Widgets/Resources/UI/qMRMLSegmentEditorWidget.ui

```diff
@@ -1,566 +1,607 @@
-   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
-   <receiver>SegmentsTableView</receiver>
-   <slot>setSegmentationNode(vtkMRMLNode*)</slot>
-   <hints>
-    <hint type="sourcelabel">
-     <x>225</x>
-     <y>12</y>
-    </hint>
-    <hint type="destinationlabel">
-     <x>239</x>
-     <y>133</y>
-    </hint>
-   </hints>
-  </connection>
-  <connection>
-   <sender>MasterVolumeIntensityMaskCheckBox</sender>
-   <signal>toggled(bool)</signal>
-   <receiver>MasterVolumeIntensityMaskRangeWidget</receiver>
-   <slot>setVisible(bool)</slot>
-   <hints>
-    <hint type="sourcelabel">
-     <x>374</x>
-     <y>454</y>
-    </hint>
-    <hint type="destinationlabel">
-     <x>179</x>
-     <y>506</y>
-    </hint>
-   </hints>
-  </connection>
-  <connection>
-   <sender>qMRMLSegmentEditorWidget</sender>
-   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
-   <receiver>SegmentsTableView</receiver>
-   <slot>setMRMLScene(vtkMRMLScene*)</slot>
-   <hints>
-    <hint type="sourcelabel">
-     <x>78</x>
-     <y>2</y>
-    </hint>
-    <hint type="destinationlabel">
-     <x>86</x>
-     <y>148</y>
-    </hint>
-   </hints>
-  </connection>
- </connections>
-</ui>
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>qMRMLSegmentEditorWidget</class>
+ <widget class="qMRMLWidget" name="qMRMLSegmentEditorWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>406</width>
+    <height>426</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>qMRMLSegmentEditorWidget</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <property name="spacing">
+    <number>4</number>
+   </property>
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
+    <number>0</number>
+   </property>
+   <item>
+    <layout class="QGridLayout" name="NodeSelectorLayout">
+     <item row="1" column="0">
+      <widget class="QLabel" name="MasterVolumeNodeLabel">
+       <property name="text">
+        <string>Master volume:</string>
+       </property>
+      </widget>
+     </item>
+     <item row="0" column="0">
+      <widget class="QLabel" name="SegmentationNodeLabel">
+       <property name="text">
+        <string>Segmentation:</string>
+       </property>
+      </widget>
+     </item>
+     <item row="0" column="1">
+      <widget class="qMRMLNodeComboBox" name="SegmentationNodeComboBox">
+       <property name="nodeTypes">
+        <stringlist>
+         <string>vtkMRMLSegmentationNode</string>
+        </stringlist>
+       </property>
+       <property name="renameEnabled">
+        <bool>true</bool>
+       </property>
+      </widget>
+     </item>
+     <item row="0" column="2">
+      <widget class="QToolButton" name="SliceRotateWarningButton">
```

Comment:
I'd specify a smaller size for the button. It makes the row higher when visible, and there is a margin around the icon that we could get rid of.

Reply:
Would you hardcode a size value in Qt designer or copy vertical size from the combobox that is in the same row?

Reply:
Probably the latter would be better. In the past I simply made them 21x21, but that doesn't work anymore.